Server/MirEnvir/Envir.cs

```diff
@@ -19,7 +19,7 @@ public class Envir
         public static object AccountLock = new object();
         public static object LoadLock = new object();
 
-        public const int Version = 53;
+        public const int Version = 55;
```

Comment:
no idea this wast me i notice it on my log as but i didn't sent this only thing was set to 55 was hum up